integration-tests/setup/setup.ts

```diff
@@ -0,0 +1,34 @@
+import { retrieveSSMParameterValue } from './retrieveSSMParameterValue'
+const region = 'eu-west-2'
+// const stackName = 'txma-query-results'
+
+// eslint-disable-next-line @typescript-eslint/prefer-namespace-keyword, @typescript-eslint/no-namespace
+declare module global {
+  const NOTIFY_MOCK_SERVER_BASE_URL: string
+}
+
+module.exports = async () => {
+  setRegionEnvVar()
+  setEnvVarFromJestGlobals()
+  await readEnvVarsFromSSM()
+}
+
+const setRegionEnvVar = () => {
```

Comment:
This can be added a a global

Reply:
Could do. Would like to know if there is an advantage of doing that. Is the region likely to ever change?